AudioDeviceExample/ViewController.swift

```diff
@@ -160,7 +248,15 @@ class ViewController: UIViewController {
         self.disconnectButton.isHidden = !inRoom
         self.disconnectButton.isEnabled = inRoom
         UIApplication.shared.isIdleTimerDisabled = inRoom
+
         if #available(iOS 11.0, *) {
+            if ((self.audioDevice as? ExampleAudioEngineDevice) != nil) {
+                self.musicButton.isHidden = !inRoom
+                self.musicButton.isEnabled = inRoom
+            }
+            self.audioDeviceButton.isHidden = inRoom
+            self.audioDeviceButton.isEnabled = !inRoom
```

Comment:
Is the button expected to always be hidden on iOS < 11.0? I think these two lines could be outside of the availability check.